src/applications/personalization/dashboard/containers/MessagingWidget.jsx

```diff
@@ -38,8 +38,12 @@ class MessagingWidget extends React.Component {
       { label: 'Date', value: 'sentDate', nonSortable: true },
     ];
 
+    // eslint-disable-next-line
```

Comment:
Why is this needed? To silent the warning about the unused `id` param?

Reply:
> Why is this needed? To silent the warning about the unused id param?

Yep


> Is this functionality going to be restored soon?
Functionality will be restored post WBC - we have work scoped out [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/13839) and other tickets.

Reply:
thanks! As long as that `eslint-disable` is removed when functionality is restored and it's no longer needed it's cool with me.